app/controllers/AMIable.scala

```diff
@@ -1,21 +1,20 @@
 package controllers
 
+import javax.inject.Inject
+
 import config.AMIableConfig
-import models.{SSA, Attempt}
+import models.{Attempt, SSA}
 import play.api._
 import play.api.libs.ws._
 import play.api.mvc._
-import prism.Prism
-import prism.PrismLogic
+import prism.{Prism, PrismLogic}
 
-import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.ExecutionContext
 
 
-class AMIable extends Controller {
+class AMIable @Inject()(ws: WSClient, playConfig: Configuration)(implicit exec: ExecutionContext) extends Controller {
```

Comment:
I may switch to the compiled version at some point, but I just wanted to upgrade to Play 2.5 with as small a change as possible. Sorry.
As we've discussed before though, I'm deeply unsatisfied with both approaches so I'm still on the fence about moving off the "simpler" (sic) and better documented default version.